index.js

```diff
@@ -20,7 +20,7 @@ module.exports = readDotEnvConfigs;
  * Application root directory.
  * @type {string}
  */
-var applicationRoot = require('app-root-path').toString();
+var applicationRoot = __dirname + '/../../';
```

Comment:
do we have to support globally installed modules? That's usually a bad practice anyways.